doc/source/serve/architecture.md

```diff
@@ -24,62 +24,91 @@ There are three kinds of actors that are created to make up a Serve instance:
   the control plane. The Controller is responsible for creating, updating, and
   destroying other actors. Serve API calls like creating or getting a deployment
   make remote calls to the Controller.
-- Router: There is one router per node. Each router is a [Uvicorn](https://www.uvicorn.org/) HTTP
+- HTTP Proxy: By default there is one HTTP proxy actor on the head node. This actor runs a [Uvicorn](https://www.uvicorn.org/) HTTP
   server that accepts incoming requests, forwards them to replicas, and
-  responds once they are completed.
-- Worker Replica: Worker replicas actually execute the code in response to a
+  responds once they are completed.  For scalability and high availability,
+  you can also run a proxy on each node in the cluster via the `location` field of `http_options`: {ref}`core-apis`
+- Worker Replicas: Actors that actually execute the code in response to a
```

Comment:
`Worker` is a somewhat overloaded term throughout Ray. Can we call these `Deployment Replicas` throughout the documentation (including the diagram) instead?

Alternatively, especially if this term is used in more than just the architecture page, we can make this change in a future PR.

Reply:
Good point, I've just changed it to call them replicas since that's what we call them elsewhere